src/components/App.tsx

```diff
@@ -15,7 +14,9 @@ const CourseDetailsPage = lazy(
 const CoursesPage = lazy(() => import("../pages/coursesPage/CoursesPage"));
 
 export const App: React.FC = () => {
-  const currentTheme = useAppSelector((state) => state.user.theme);
+  const LocalStorageTheme = localStorage.getItem("theme");
+  const storeTheme = useAppSelector((state) => state.user.theme);
```

Comment:
It's not really important now, but I think all selectors should be functions in a separate file that is located next to the slice that uses this selector.